### ISSUE TITLE
k8s-rsgen

### ISSUE THREAD

**Issue by andrenth** [camallo/k8s-client-rs#2]
Hi

Are you still working on this library? I tried to follow the intructions on `api-specs/README.md` to build for k8s v1.8, but there's no `api-specs/skeleton`, and I couln't find anything about `k8s-rsgen`.

Thanks,
Andre

**Comment by lucab** [comment]
No, I'm not working anymore on this library. I think I may have some uncommitted files, as I didn't reach a working state within a reasonable time. If you are interested in pushing this forward, I can spend some time to re-sync the code and add you as a committer.

**Comment by andrenth** [comment]
Hi Luca

Honestly I'm too new to both Rust and Kubernetes to take over the project, but if you could add the missing files, I'd like to give it a try.

Cheers.

**Comment by lucab** [comment]
I pushed generator, templates and initial generated code at https://github.com/camallo/k8s-client-rs/compare/wip/k8s-apigen?expand=1. But I don't expect this to be in a clean nor usable state.
